src/Utils/dialogEditing.ts

```diff
@@ -61,11 +61,14 @@ export async function onInsertAction(
     // Find top scoring Action
     let insertedAction = DialogUtils.getBestAction(uiScoreResponse.scoreResponse, actions, canEndSession)
 
-    // None were qualified so pick the first (will show in UI as invalid)
+    // None were qualified so try to find a valid one in unscored action
```

Comment:
nit : suggest moving this comment inside the `if` statement since it only applies if the condition is met.

Reply:
My pref is for comment above if for easier reading, but I changed the comment to clarify:
 // If none were qualified, try to find a valid one in unscored action